src/_scss/pages/search/searchSidebar.scss

```diff
@@ -3,64 +3,19 @@
     @import '../../elements/_inputs';
     @import '../../elements/_labels';
 
+    .sidebar-header {
+        margin-left: 1.5rem;
+    }
+
     .filter-header {
         font-size: $base-font-size;
-        font-weight: bold;
+        font-weight: normal;
         padding-bottom: 1rem;
-        padding-left: 1rem;
         display: inline-block;
+        line-height: auto;
     }
 
-	.search-option {
-		margin-bottom:1rem;
-
-        label {
-            @include usa-label;
-        }
-
-        svg {
-            fill: $color-gray;
-            height: 2rem;
-            width: 2rem;
-            position:relative;
-            cursor: pointer;
-        }
-
-        .toggle {
-            @include button-unstyled;
-            display: inline-block;
-            vertical-align: middle;
-
-            &.closed {
-
-            }
-
-            &.open {
-
-            }
-
-            &.hide {
-                display: none;
-            }
-
-            span {
-                // this is for the animation group
-                margin: 0;
-                top: 0;
-            }
-        }
-
-        .search-filter {
-            padding-left: 1.5rem;
-            padding-bottom: 3rem;
-            border-bottom: 1px solid $color-gray-lighter;
-
-            .filter-header {
-                padding-left: 0;
-            }
-        }
-	}
-
     @import "./_filters";
+    @import "./searchOption";
```

Comment:
My personal approach is to use `_` prefixes for files that support a section (a file that just deals with positioning as `_positioning.scss` for example) while no `_` for files that are the "entrypoint" when being imported by external files (`sidebar` for example, which in turn imports `_positioning.scss`).